### DISCUSSION TITLE
Good design, lots of bugs

### DISCUSSION THREAD

**Discussion by serzhan181** [heroui-inc/heroui#971] [Feedback]
I like this UI library, however it has some problems with next js, hydrations in particular. It looks good... until u open console and see huge error logs. Most of them are hydration errors as i said. To fix that, i needed to create my own components. 3/5 overall

**Comment by jrgarciadev** [comment]
Thanks for the feedback, we are trying all the hydration issues, but it not only depends on the NextUI team since the issue comes from the `react-aria` library which is the `behaviour` / `a11y` library we use in order to build high-quality components, they only happen on `strict` mode since `react-aria` doesn't support `strict` mode with `react-18`, contributions are always welcome 🙌🏻
